kcli_plan.yml

```diff
@@ -23,10 +22,14 @@
   - 01_create_iso.sh
   - 02_install_baremetal.sh
   - sno_params.yml
-  - origin: helpers/racadm.sh
+  - origin: racadm.sh
     path: /root/bin/racadm.sh
   - origin: {{ pull_secret }}
     path: /root/openshift_pull.json
   scripts:
   - deploy.sh
-
+{% if installer_wait %}
+  finishfiles:
```

Comment:
what's finishfiles?
anyhow, can plz be a separate commit?

Reply:
moved to https://github.com/yuvalk/kcli-sno/pull/2